tests/unit_tests/analysis/test_es_update.py

```diff
@@ -801,8 +801,9 @@ def g(X):
 
     prior_da = prior.load_parameters(param_group, range(num_ensemble))
     posterior_da = posterior_ens.load_parameters(param_group, range(num_ensemble))
-    # Because of adaptive localization, not all parameters should be updated.
-    # This would fail if with global updates.
+    # Make sure some, but not all parameters were updated.
+    assert not np.isclose(prior_da, posterior_da).all()
```

Comment:
is it the same to `np.allclose()`? 
https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
Yes, those are the same thing. I've pushed a fix. Thanks 👍